mock-core-configs/etc/mock/templates/navy-8.tpl

```diff
@@ -0,0 +1,69 @@
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config system-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['dist'] = 'el8'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '8'
+config_opts['package_manager'] = 'yum'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['macros']['%dist'] = '.el8'
+config_opts['bootstrap_image'] = 'navylinux:8'
+config_opts['macros']['vendor'] = "The Navy Linux Project."
+config_opts['macros']['packager'] = "packager@navylinux.org"
+config_opts['macros']['%releasever'] = "8"
+
+
+config_opts['dnf.conf'] = """
+[main]
+keepcache=1
+debuglevel=2
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=mock
+syslog_device=
+mdpolicy=group:primary
+best=1
+protected_packages=
+module_platform_id=platform:el8
+user_agent={{ user_agent }}
+
+[nl-base]
+name=Navy Linux Enterprise OS Repository - el$releasever
+baseurl=https://cdn.navylinux.org/navylinux/releases/8.4r1/x86_64/os/
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-navy-linux-official
```

Comment:
This file does not exist on my workstation, so the build will fail. There are several ways how to handle it. The easiest one is to add the key to https://github.com/xsuchy/distribution-gpg-keys/ and use the key from that package (every other config does that).

Reply:
Ok, I will follow up thanks for the help.